content/articles/how-to-build-a-real-time-location-tracker-using-leaflet-js/index.md

```diff
@@ -0,0 +1,444 @@
+            height: 100vh;
+        }
+    </style>
+</head>
+
+<body>
+    <div id="map"></div>
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
+        integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
+        crossorigin=""></script>
+    <script src="https://unpkg.com/leaflet-control-geocoder/dist/Control.Geocoder.js"></script>
+    <script>
+        var map_init = L.map('map', {
+            center: [9.0820, 8.6753],
+            zoom: 8
+        });
+        var osm = L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+        }).addTo(map_init);
+        L.Control.geocoder().addTo(map_init);
+    </script>
+</body>
+
+</html>
+```
+
+#### Location tracker
+HTML5 geolocation is regarded as the best HTML5 API. With permission, it identifies the user's geographic information which can be sent to a backend server for location-conscious processes like showing products accessible in geographic zones close to the users.
+
+In this section, we will show you how to build a location tracker that pings users' geographic information every five seconds. This information includes latitude, longitude, and accuracy (closeness of the user's device from the true position).
+
+#### Handling errors
+
+```js
+if (!navigator.geolocation) {
+  console.log("Your browser doesn't support geolocation feature!");
+} else {
+  navigator.geolocation.getCurrentPosition(getPosition);
+}
+```
+
+`navigator.geolocation` is a global object that returns a geolocation object by the browser that gives web content access to the location of the device. Most browsers are geolocation enabled. 
+
+Browsers like internet explorer only support HTML5 geolocation 9.0 and above. Therefore the aim of the conditional statement above is to handle such errors. 
+
+The `getCurrentPosition` method gives the current geographic information of the device.
+
+#### Getting geographic information
+
+```js
+var marker, circle, lat, long, accuracy;
+
+function getPosition(position) {
+  // console.log(position)
+  lat = position.coords.latitude;
+  long = position.coords.longitude;
+  accuracy = position.coords.accuracy;
+
+  if (marker) {
+    map_init.removeLayer(marker);
+  }
+
+  if (circle) {
+    map_init.removeLayer(circle);
+  }
+
+  marker = L.marker([lat, long]);
+  circle = L.circle([lat, long], { radius: accuracy });
+
+  var featureGroup = L.featureGroup([marker, circle]).addTo(map_init);
+
+  map_init.fitBounds(featureGroup.getBounds());
+
+  console.log(
+    "Your coordinate is: Lat: " +
+      lat +
+      " Long: " +
+      long +
+      " Accuracy: " +
+      accuracy
+  );
+}
+```
+
+We earlier said that the callback method `getCurrentPosition` gives the current geographic location of users. To view this information's, we will "`console.log`" the function's argument. 
```

Comment:
The second sentence here requires a word to make sense. Kindly check on it's readability.